docs/source/getting-started/install.rst

```diff
@@ -209,35 +207,87 @@ Installation on Windows
 
 - Press enter. This will configure your ScanCode installation.
 
-- Several messages are displayed followed by the scancode command help.
+- Several messages are displayed followed by the ScanCode command help.
 
 - The installation is complete.
 
+This uses the default Python present in the PATH environment variable i.e. the last Python
+installed registers itself in the environment is the default. You can also use the ``configure``
+script to explicitly provide the Python path to ScanCode.
+
+- Follow the Instructions above till changing the current location of your command prompt to the
+  root directory where ScanCode is installed.
+
+- Run this command with the path to Python Executable::
+
+    configure --python path/to/python
+
+- You can also use ``path`` instead of ``path/to/python`` to use the python from PATH environment
+  variable. More information is available at the `configure <https://github.com/nexB/scancode-toolkit/blob/develop/configure.bat>`_ script (L6-L15).
+
+- Now you can run ``scancode -h`` to display the Help Text, and here the installation is complete.
+
 Un-installation
 ^^^^^^^^^^^^^^^
 
 - Delete the directory in which you extracted ScanCode.
 - Delete any temporary files created in your system temp directory under a ScanCode directory.
 
----
+----
 
-.. _source_configure_install:
+.. _pip_install:
 
-Build From Source
------------------
+Installation as a library: via ``pip``
+--------------------------------------
+
+ScanCode can be easily installed using ``pip``. The steps are:
+
+#. Create a Python 3.6 Virtual Environment::
+
+    virtualenv -p /usr/bin/python3.6 venv-scancode
+
+For more information on Python virtualenv, visit this `page <https://docs.python-guide.org/dev/virtualenvs/#lower-level-virtualenv>`_.
+
+#. Activate the Virtual Environment you just created::
+
+    source venv-scancode/bin/activate
+
+#. Run ``pip install scancode-toolkit`` to install the latest version of Scancode.
+
+.. NOTE::
+
+    If you use Python 2.7, scancode-toolkit Version 3.0.2 is installed by default. For Python 3
+    the latest version of Scancode Toolkit is installed by default. Requesting a specific version
+    through ``pip install`` for Python 3 will give Errors if the Version isn't 3.1.x or later.
+
+.. WARNING::
+
+    Python 3.7.x and 3.8.x is not supported yet.
+
+To uninstall, run ``pip uninstall scancode-toolkit``.
+
+----
+
+.. _source_code_install:
+
+Installation from Source Code: Git Clone
+----------------------------------------
 
 You can also download the Scancode Toolkit Source Code and build from it yourself. This is how you
 would want to do it if:
 
-- You are Adding new patches to Scancode and want to test it.
-- You want to test a specific Version/Checkpoint/Branch from the VCS
+- You are Adding new patches to Scancode and want to test it. So you build ScanCode locally
+  with your added changes.
 
+- You want to test a specific Version/Checkpoint/Branch from the VCS.
 
```

Comment:
I think we should move this section after Download the ScanCode-Toolkit Source Code section

Reply:
@AyanSinhaMahapatra This is the final change we need; then this is ready to go.

Reply:
@AyanSinhaMahapatra This is the final change we need; then this is ready to go.